build.assets/charts/Dockerfile-distroless

```diff
@@ -30,4 +30,7 @@ FROM $BASE_IMAGE
 COPY --from=teleport /opt/staging /
 COPY --from=staging /opt/staging/root /
 COPY --from=staging /opt/staging/status /var/lib/dpkg/status.d
+# Attempt a graceful shutdown by default
+# See https://goteleport.com/docs/reference/signals/ for signal reference.
+STOPSIGNAL SIGQUIT
```

Comment:
I think we can remove this in the future.
https://github.com/gravitational/teleport/blob/d88e54907062d43c391ca6b4b7d768b6deae9103/lib/integrations/awsoidc/deployservice.go#L494-L503

Reply:
There might be other implications to having `teleport` be the init in the container pid namespace as opposed to a known-good pid 1 implementation, but we could/should certainly remove the signal rewrite.